### DISCUSSION TITLE
Using Main for PRs instead of Development.

### DISCUSSION THREAD

**Discussion by paganotoni** [gobuffalo/cli#132] [Ideas]
Now that Go modules have generalized I wonder if we should keep requesting PR's to development. I remember that `master` branch needed to be untouched because before Go modules the latest commit was what applications used to build their apps, however, now that we have modules applications no longer use our main branch as the base for their development and hence it could be moving as we merge PR's without affecting the compilation/use of Buffalo, unless they explicitly update to the version/commit in main.

The main benefit I see from moving to use main instead of development is that we would reduce one step from our release process. I have not thought about any negative effect besides GOPATH based development but would love to read your thoughts.

cc @gobuffalo/all @sio4 @fasmat

**Comment by sio4** [comment]
Totally agree. Actually, that is what I want to say but just postponed until we start discussing v1.

Not directly related, one another idea of mine is that I think just making the current version as v1 in near future after polishing it if there are important issues to be fixed, freezing them unless any security bug, and start new jump as v2 and focusing all our effort on the new version would be a most feasible strategy for us.

**Comment by fasmat** [comment]
I'm fine with dropping the development branch in favor of working directly on main. The only drawback I see is that then it might be harder to create release notes for every new version, but maybe we can do that also based on the diff between the latest tag and current instead of main and develop?

**Reply by sio4** [reply]
If I understood correctly, the automated note made by github may not care if those PRs were merged in to the main directly or via another branch. I think that will be fine.

**Comment by sio4** [comment]
### Conclusion of the Discussion

Congratulations! The actions were taken and now we deprecated the `development` branch for both Buffalo core and CLI!
